tuf/client/client_test.go

```diff
@@ -250,15 +251,25 @@ func TestChecksumMismatch(t *testing.T) {
 
 	sampleTargets := data.NewTargets()
 	orig, err := json.Marshal(sampleTargets)
-	origSha256 := sha256.Sum256(orig)
 	assert.NoError(t, err)
 
+	origHashes := GetHashes(orig)
+
 	remoteStorage.SetMeta("targets", orig)
 
-	_, _, err = client.downloadSigned("targets", int64(len(orig)), origSha256[:])
+	_, _, err = client.downloadSigned("targets", int64(len(orig)), origHashes)
 	assert.IsType(t, ErrChecksumMismatch{}, err)
 }
 
+func GetHashes(raw []byte) data.Hashes {
```

Comment:
It is ugly here, will use a common function for all the other code who want to generate the `hashes`